frontend/ui/widget/virtualkeyboard.lua

```diff
@@ -736,7 +736,7 @@ local VirtualKeyboard = FocusManager:new{
     umlautmode = false,
     layout = {},
 
-    width = Screen:scaleBySize(600),
+    width = nil, -- non-fullscreen keyboard may cause design issues (eg Text editor)
```

Comment:
But Text editor is _not_ non-fullscreen. What do you mean ? Which design issues ?

Reply:
https://github.com/koreader/koreader/blob/8ec60f1203b8b7bcaad37a5d5149bd109154fa5f/frontend/ui/widget/virtualkeyboard.lua#L779
I mean that if a developer wants to show non-fullwidth keyboard somewhere, he must check the design carefully.
I tried to show non-fullwidth keyboard in Text editor and it ugly shows lower widgets at the edges.
I would remove the comment to avoid confusing.

Reply:
Ah, ok, non-fullscreen _keyboard_, that we never meet.
If it's just ugly and make no real sense, I think you can remove the width option and hardcode it to Screen.width.
Or just keep your comment, now that I understand it :)